e2e/test/iothub/service/RegistryManagerExportDevicesTests.cs

```diff
@@ -47,14 +47,15 @@ public class RegistryManagerExportDevicesTests : E2EMsTestBase
         [Timeout(120000)]
         [DoNotParallelize]
         [DataRow(StorageAuthenticationType.KeyBased, false)]
-        [DataRow(StorageAuthenticationType.IdentityBased, false)]
-        [DataRow(StorageAuthenticationType.IdentityBased, true)]
+        //[DataRow(StorageAuthenticationType.IdentityBased, false)]
```

Comment:
Can you elaborate why these are commented out?

Reply:
Accidentally included in the PR. Thanks for catching. They were making it awkward to debug so I (meant to) temporarily disable them.